tests/unit/Actions.spec.ts

```diff
@@ -55,37 +58,14 @@ const nrData = {
 
 describe('Action button states', () => {
   let wrapper: any
-  let setEntityType: any
-  let setChanged: any
-  let setValidity: any
-  let setEditing: any
 
   beforeAll(async () => {
     // initialize store
-    store.state.stateModel.tombstone.filingType = 'correction'
-    store.state.stateModel.correctionInformation.type = 'CLIENT'
+    store.stateModel.tombstone.filingType = FilingTypes.CORRECTION
+    store.stateModel.correctionInformation.type = CorrectionErrorTypes.CLIENT
```

Comment:
I think we shouldn't use types/IFs or enums in the unit tests, so that they are more fragile, and so that an error in a type or enum isn't cancelled out in the unit tests.